app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -871,6 +871,59 @@ def update_setting
     redirect_to exchanges_hbx_profiles_root_path
   end
 
+  # GET Endpoint to fetch family information for presenting the information in the UI.
+  # This endpoint is used to display family information for QAs to review the creates and updates made to the Family, Family Members, and People.
+  # The endpoint is accessible only to users with the `can_view_dry_run_dashboard?` permission.
+  #
+  # @param [String] id The ID of the family or person to fetch information for.
+  # @return [JSON] A JSON object containing family and family member information.
+  # @example
+  #   GET /exchanges/hbx_profiles/raw_family_information?id=12345
+  # @note This endpoint is intended for internal use by QAs in the lower environments to verify the correctness of the family data.
+  def raw_family_information
+    authorize HbxProfile, :can_view_dry_run_dashboard?
```

Comment:
not huge, but in case these ever need differing policies should we have a new policy for this endpoint?

Reply:
that's fine then!